experimental/material_3_demo/lib/typography_screen.dart

```diff
@@ -0,0 +1,63 @@
+// Copyright 2021 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+class TypographyScreen extends StatelessWidget {
+  const TypographyScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final textTheme = Theme.of(context)
+        .textTheme
+        .apply(displayColor: Theme.of(context).colorScheme.onSurface);
+    return Expanded(
+      child: ListView(
+        children: <Widget>[
+          const SizedBox(height: 7),
+          TextStyleExample(
```

Comment:
This would be easier to read if all of the TextStyleExample calls were one-liners.